app/models/organisation.rb

```diff
@@ -0,0 +1,5 @@
+class Organisation < ApplicationRecord
+  has_many :users
+
+  validates :name, presence: true
```

Comment:
I've kept the validation really simple at the moment. It's worth having some as it makes testing errors a little easier.